### PR TITLE
Update Grafana Dashboard

### PR DESCRIPTION
**Pull request by NavinSharma13**
## Description
 - Update grafana image tag from default to v9.1.4
 - Add `Data Source` variable in dashboard panel to filter by datasource.
 - Update metrics for old metrics which were missing
 -  Add new panels for metrics 
```
    metrics_com_linkedin_metadata_kafka_MetadataChangeLogProcessor_EntityChangeEventGeneratorHook_latency
    metrics_com_linkedin_metadata_kafka_MetadataChangeLogProcessor_IngestionSchedulerHook_latency
    metrics_com_linkedin_metadata_kafka_MetadataChangeLogProcessor_MetadataTestHook_latency
    metrics_com_linkedin_metadata_kafka_MetadataChangeLogProcessor_NotificationGeneratorHook_latency
    metrics_com_linkedin_metadata_kafka_MetadataChangeLogProcessor_SiblingAssociationHook_latency
    metrics_com_linkedin_metadata_kafka_MetadataChangeLogProcessor_UpdateIndicesHook_latency
```
     
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by szalai1 — APPROVED** [review]
LGTM

File: docker/monitoring/docker-compose.consumers.monitoring.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-version: '3.5'
```
Comment (szalai1) [inline docker/monitoring/docker-compose.consumers.monitoring.yml:2]: why was this needed to bump?

Comment (NavinSharma13) [reply docker/monitoring/docker-compose.consumers.monitoring.yml:2]: to make it compatible with other docker-compose file under [datahub](https://github.com/datahub-project/datahub)/[docker](https://github.com/datahub-project/datahub/tree/master/docker)/docker-compose-without-neo4j.yml
